README.md

```diff
@@ -5,18 +5,15 @@ to help developers get their hands on a fully functional and easy to understand
 
 The tutorial can be found [here](https://docs.login.xyz).
 
-## Running the Example
+## Getting Started
 
-Add the following to `.env.local.example` in order to run the application:
+Make a copy of `.env.local.example` to `.env.local`, and fill in the following two values. For example:
 
 ```
+NEXTAUTH_URL=http://localhost:3000
 NEXTAUTH_SECRET=somereallysecretsecret
-JWT_SECRET=itshouldbealsoverysecret
-DOMAIN=localhost:3000
```

Comment:
if we remove those two lines from the config, the example won't run as expected. no sign out button will be displayed after login. @w4ll3 can you check what the reason for this is?

Reply:
It works for me, couldn't reproduce on Linux nor Mac.